.changeset/dry-trainers-own.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-extension-dsl-data-space-studio': patch
+'@finos/legend-extension-dsl-data-space': patch
+---
+
+Added form and state management features to DataSpace and FormTextEditor components.
```

Comment:
FormTextEditor should be renamed to DataSpaceEditor

Reply:
Done